# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/noinstalllicense.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-256.2/meson.build.vanilla	2024-07-12 13:36:35.339032036 +0200
+++ systemd-256.2/meson.build	2024-07-12 13:36:53.716031955 +0200
@@ -2729,8 +2729,8 @@
              'docs/UIDS-GIDS.md',
              install_dir : docdir)
 
-install_subdir('LICENSES',
-               install_dir : docdir)
+#install_subdir('LICENSES',
+#               install_dir : docdir)
 
 install_emptydir(systemdstatedir)
 
